corehq/apps/reports/static/reports/js/base_maps.js

```diff
@@ -0,0 +1,12 @@
+hqDefine("reports/js/base_maps.js", function() {
+    var initial_page_data = hqImport("hqwebapp/js/initial_page_data.js").get;
+    CONTEXT = initial_page_data('context');
```

Comment:
Can we define these locally inside this `hqDefine` block, or are they accessed globally? If global (yuck!), would be nice to mark the globals at the top of the file.